packages/testing/index.js

```diff
@@ -18,4 +19,5 @@ export {
   elementUpdated,
 } from '@open-wc/testing-helpers/index.js';
 
-export { chai, expect, should, assert } from '@bundled-es-modules/chai';
+// @ts-ignore
+export const { expect, should, assert } = window.chai;
```

Comment:
does this work well with type checking? or is that solved in the other PR?

Reply:
yeah I wanna rebase the typing branch on this one and see if it "works/helps" before merging this

Reply:
Ok, LGTM otherwise so just merge whenever.